### ISSUE TITLE
(400 Bad Request) for update_token

### ISSUE THREAD

**Issue by EineKleinee** [proshunsuke/colmsg#109]
All Message Apps have updated their request headers after a force update. If the changes are not applied, 400 Bad Request error will occur. I would greatly appreciate it if you could update this when you have free time. Thank you so much!

Details are as follows:

**x-talk-app-id**
```
jp.co.sonymusic.communication.keyakizaka 2.4
jp.co.sonymusic.communication.nogizaka 2.4
jp.co.sonymusic.communication.sakurazaka 2.4
jp.co.sonymusic.communication.asukasaito 2.4
jp.co.sonymusicsolutions.mizukiyamashita 2.4
```

User-Agent has also changed, but it seems that updating the UA or not does not affect the requests:

**user-agent**
`Dart/3.4 (dart:io)`

**Comment by axlchr12** [comment]
Yup, just update the `x-talk-app-id`; the `User-Agent` doesn’t affect anything. I’m using `Hot/2.16.01 (jp.co.sonymusic.communication.keyakizaka; build:224; iOS 18.2.0) Alamofire/5.8.0` as the `User-Agent`, and it still works.

![image](https://github.com/user-attachments/assets/c82c1fba-b79c-4536-9d47-adb6020eed0e)

**Comment by proshunsuke** [comment]
The issue itself has been resolved, but there is a problem with the release build. Please wait a little longer.

**Comment by axlchr12** [comment]
> The issue itself has been resolved, but there is a problem with the release build. Please wait a little longer.

Any info for release build?

**Comment by proshunsuke** [comment]
released v3.2.1
https://github.com/proshunsuke/colmsg/releases/tag/v3.2.1


I struggled with the release process due to changes in the build environment compared to previous.
Please let me know if you encounter any issues.

**Comment by axlchr12** [comment]
> released v3.2.1 https://github.com/proshunsuke/colmsg/releases/tag/v3.2.1
> 
> I struggled with the release process due to changes in the build environment compared to previous. Please let me know if you encounter any issues.

![image](https://github.com/user-attachments/assets/0923bac7-61ac-4df9-95bd-ca8743210bc8)
![image](https://github.com/user-attachments/assets/64737308-dd4e-44c4-a4b9-d1f18d1c01f2)
![image](https://github.com/user-attachments/assets/07c07582-7469-42ef-9d71-a0b805c3940c)


It works as usual. Thank you! You may close this issue, as it has been resolved in the recent release build.
